generator_vllm.py

```diff
@@ -42,92 +46,69 @@ def generate(prompt, max_retries=20):
     retries = 0
     while retries < max_retries:
         try:
-            response = requests.post(API_ENDPOINT, json=payload, headers={'Authorization': f'Bearer {API_KEY}'}, timeout=120)
+            response = requests.post(
+                API_ENDPOINT, json=payload, headers={"Authorization": f"Bearer {API_KEY}"}, timeout=120
+            )
             response.raise_for_status()
             result = response.json()
-            print(result)
-            return result['choices'][0]['message']["content"].strip()
+            return result["choices"][0]["message"]["content"].strip()
         except (requests.RequestException, KeyError) as e:
             print(f"Error: {e}")
             retries += 1
-            time.sleep(10)  # wait for 2 seconds before retrying
+            time.sleep(2)  # wait for 2 seconds before retrying
     return "Failed to generate response after several attempts."
 
+
 def process_batch(batch):
-    print(batch)
+    print(batch.shape, batch.columns)
```

Comment:
p3: I think you can remove this print statement.

Reply:
OK